recipes/cpython/all/conandata.yml

```diff
@@ -1,7 +1,7 @@
 sources:
-  "3.10.0":
-    url: "https://www.python.org/ftp/python/3.10.0/Python-3.10.0.tgz"
-    sha256: "c4e0cbad57c90690cb813fb4663ef670b4d0f587d8171e2c42bd4c9245bd2758"
+  "3.10.3":
+    url: "https://www.python.org/ftp/python/3.10.3/Python-3.10.3.tgz"
+    sha256: "5a3b029bad70ba2a019ebff08a65060a8b9b542ffc1a83c697f1449ecca9813b"
```

Comment:
Please don't remove older versions from the most recent minor version.
We will remove support for older minor releases when cpython is on 3.11 (like the cmake recipe does).